### PR TITLE
Update client.py for python3

### PR DESCRIPTION
**Pull request by MichaelStadtmiller**
the package was breaking for me on line 182. The fix seemed to be adding parens to a print statement on line 182

### CODE REVIEW

File: petfinder/client.py [hunk 1]
```diff
@@ -179,7 +179,7 @@ def _parse_pet_record(self, root):
             # copy to the record as key/val.
             node = root.find(field)
             if node is None:
-                print "SKIPPING %s" % field
+                print ("SKIPPING %s") % field
```
Comment (gtaylor) [inline petfinder/client.py]: Looks like there's a space between print and the opening parenthesis. I'd also prefer that the substitution happen within the parens:
```
print ("SKIPPING %s" % field )
```

**Review by gtaylor — CHANGES_REQUESTED** [review]
Thanks for sending this in! One minor style nit. If you can fix then squash, I'll get this merged.

**Comment by MichaelStadtmiller** [comment]
OK thanks I made the changes and committed.

**Comment by gtaylor** [comment]
Thanks!

**Pull request merged**